sql/core/src/main/scala/org/apache/spark/sql/sources/commands.scala

```diff
@@ -350,6 +356,7 @@ private[sql] abstract class BaseWriterContainer(
     }.getOrElse {
       // If output committer class is not set, we will use the one associated with the
       // file output format.
+      logInfo(s"Using output committer class ${outputFormatClass.getCanonicalName}")
```

Comment:
Maybe we can get the instance of `outputCommitter` first and then log its class?
```
val outputCommitter = ...
logInfo(...)
outputCommitter
```